src/components/BaseLayerSwitcher/BaseLayerSwitcher.js

```diff
@@ -109,6 +112,14 @@ function BaseLayerSwitcher({
 
   const nextImage = getNextImage(currentLayer, baseLayers, images);
 
+  /* Move visible layer to front of array (array.sort() not working in Firefox) */
```

Comment:
Array.sort is an "in place" operation so you shouldn't chain it with a `.map()` call. Can you try to replace this code block with `baseLayers.sort(...)`?

Reply:
I now managed to fix sort(), Mozilla's behaviour may turn out inconsistent if the returned value in the compareFunction is 0 (stated in MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort), but in our case we can use 1 instead